server/routes/api/choropleth.py

```diff
@@ -156,48 +149,48 @@ def choropleth_geo():
         # Valid response needs at least geometry and a name.
         if json_text and geo_id in names_by_geo:
             geo_feature = {
-                "type": "Feature",
-                "geometry": {
-                    "type": "MultiPolygon",
-                },
-                "id": geo_id,
-                "properties": {
-                    # Choose the first name when multiple are present.
-                    "name": names_by_geo.get(geo_id, ["Unnamed Area"])[0],
-                    "hasSublevel": (display_level in LEVEL_MAP),
-                    "geoDcid": geo_id,
+                    "type": "Feature",
+                    "geometry": {
+                        "type": "MultiPolygon",
+                    },
+                    "id": geo_id,
+                    "properties": {
+                        # Choose the first name when multiple are present.
+                        "name": names_by_geo.get(geo_id, ["Unnamed Area"])[0],
+                        "hasSublevel": 
+                            (display_level in LEVEL_MAP),
+                        "geoDcid": geo_id,
+                    }
                 }
-            }
             # Load, simplify, and add geoJSON coordinates.
             # Exclude geo if no or multiple renderings are present.
             if len(json_text) != 1:
                 continue
             geojson = json.loads(json_text[0])
             geo_feature['geometry']['coordinates'] = (
-                coerce_geojson_to_righthand_rule(geojson['coordinates'],
-                                                 geojson['type']))
-            # Process Statistical Observation if valid.
-            if ('data' in population_by_geo.get(geo_id, []) and
-                    population_by_geo[geo_id]['data']):
-                # Grab the latest available data.
-                latest_data = get_latest_data(population_by_geo[geo_id])
-                if latest_data:
-                    geo_feature["properties"]["pop"] = latest_data
+                coerce_geojson_to_righthand_rule(
+                                    geojson['coordinates'],
+                                    geojson['type']))
+
+            data = get_data(population_by_geo[geo_id])
+            # TODO(edumorales): return all populations
+            # get the selectedDate population in front-end.
```

Comment:
What do you consider the front-end (python code?) this comment is redundant since then (or drop the "in front-end")

Reply:
fixed.

Reply:
fixed.